### DISCUSSION TITLE
提示“接口响应错误 - Too Many Requests”

### DISCUSSION THREAD

**Discussion by wade657** [yetone/bob-plugin-openai-polisher#27] [Q&A]
请问这个如何处理呢？使用的不多，应该没到使用限额，尝试了各种方法都不行。多谢！
![Xnip2023-06-02_10-41-14](https://github.com/openai-translator/bob-plugin-openai-polisher/assets/135347997/52d63667-e7a7-4ae8-94fa-e5dda2e3e7e5)

**Comment by Babyfacer** [comment]
我也是。更新到了最新版本就出現此錯誤。我用的是 GPT 3.5。有國家 IP 限制嗎？

**Comment by Opheliayq** [comment]
Same here. 我也是一模一样碰到这个问题
